cla_backend/apps/cla_butler/__init__.py

```diff
@@ -0,0 +1 @@
+# -*- coding: utf-8 -*-
```

Comment:
My editor does this by default. I don't mind this in there but I can remove it if you want